azext_iot/central/_help.py

```diff
@@ -316,6 +316,20 @@ def _load_central_monitors_help():
             az iot central app validate-messages --app-id {app_id} -d {device_id} --cg {consumer_group_name}
     """
 
+    helps[
+        "iot central app monitor-properties"
+    ] = """
+        type: command
+        short-summary: Monitor desired and reported  sent to/from the IoT Hub for an IoT Central app.
+        long-summary: |
+                    EXPERIMENTAL requires Python 3.5+
```

Comment:
Is this command really experimental? The use of experimental in the monitoring commands in the IoT extension has historically been more due to the reliance of a C extension for amqp and managing it out-of-band compared to typical python package dependencies. Maybe associating this as a preview command is good enough?